### PR TITLE
[VEN-2499][VEN-2504]: XVSVault configuration on Arbitrum

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #VEN-2499 and #VEN-2504

### CODE REVIEW

File: multisig/proposals/arbitrumsepolia/vip-004/index.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      target: XVS_STORE,
+      signature: "acceptAdmin()",
+      params: [],
+    },
+
+    {
+      target: arbitrumsepolia.XVS_VAULT_PROXY,
+      signature: "setAccessControl(address)",
+      params: [ACM],
+    },
```
Comment (GitGuru7) [inline multisig/proposals/arbitrumsepolia/vip-004/index.ts:27]: ACM already configured, we can remove this command

Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-004/index.ts]: Done https://github.com/VenusProtocol/vips/pull/250/commits/c90febea2255ecd32f1f7bc707fef9af85232b9f

File: multisig/proposals/arbitrumsepolia/vip-004/index.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        "setWithdrawalLockingPeriod(address,uint256,uint256)",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+
+    {
+      target: arbitrumsepolia.XVS_VAULT_PROXY,
+      signature: "add(address,uint256,address,uint256,uint256)",
+      params: [arbitrumsepolia.XVS, 100, arbitrumsepolia.XVS, "0", 604800],
+    },
```
Comment (chechu) [inline multisig/proposals/arbitrumsepolia/vip-004/index.ts:75]: I would add the command to pause the vault. On mainnet, the vault will be paused for some days, to allow users to bridge XVS from other networks. Some days later, Prime will be enabled and the vault will be enabled, and users will be able to stake their XVS, starting the qualification period for Prime at the same time.

That was the process followed for Ethereum mainnet. By pausing the XVSVault initially, we'll be able to test the UI with this paused status

Comment (chechu) [reply multisig/proposals/arbitrumsepolia/vip-004/index.ts:75]: See [this](https://github.com/VenusProtocol/vips/blob/develop/multisig/proposals/ethereum/vip-004/index.ts#L67) as a reference

Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-004/index.ts:75]: Got it. Added the command: https://github.com/VenusProtocol/vips/pull/250/commits/e9896a8fa19d150d8444164ea4a866ed95e8c8b5

**Review by chechu — APPROVED** [review]

**Comment by Debugger022** [comment]
Multisig tx Arbitrum Sepolia: https://sepolia.arbiscan.io/tx/0x2e6e699377ef27188d6ca249c97360f810d308e39b9a3a8b853f46a6d28c5d34

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/319?chainId=56

**Pull request merged**